Watchman.Engine/Generation/Sqs/SqsResourceAlarmGenerator.cs

```diff
@@ -84,24 +77,35 @@ public class SqsResourceAlarmGenerator : IResourceAlarmGenerator<QueueDataV2, Sq
             AlertingGroupParameters groupParameters,
             AwsResource<QueueDataV2> entity)
         {
-            var expanded = await _builder.CopyAndUpdateDefaultAlarmsForResource(entity, defaults, service, resource);
+            var mergedConfig = AlarmHelpers.MergeServiceAndResourceConfiguration(service.Options, resource.Options);
```

Comment:
This feels like it should be a method on the service rather than it exposing it's internals just to be passed into a helper method.